Formula/youtube-viewer.rb

```diff
@@ -0,0 +1,147 @@
+class YoutubeViewer < Formula
+  desc "Lightweight app for searching and streaming videos from YouTube"
+  homepage "https://trizenx.blogspot.com/2012/03/gtk-youtube-viewer.html"
+  url "https://github.com/trizen/youtube-viewer/archive/3.5.6.tar.gz"
+  sha256 "b4d2d17e3ff8aba5f3c358db170867d89ca5e35f3340312c81b9af7c70f3c826"
+  head "https://github.com/trizen/youtube-viewer.git"
+
+  # build deps
+  resource "inc::latest" do
+    url "https://cpan.metacpan.org/authors/id/D/DA/DAGOLDEN/inc-latest-0.500.tar.gz"
+    sha256 "daa905f363c6a748deb7c408473870563fcac79b9e3e95b26e130a4a8dc3c611"
+  end
+
+  resource "TAP::Harness" do
+    url "https://cpan.metacpan.org/authors/id/L/LE/LEONT/Test-Harness-3.42.tar.gz"
+    sha256 "0fd90d4efea82d6e262e6933759e85d27cbcfa4091b14bf4042ae20bab528e53"
+  end
+
+  resource "CPAN::Meta" do
+    url "https://cpan.metacpan.org/authors/id/D/DA/DAGOLDEN/CPAN-Meta-2.143240.tar.gz"
+    sha256 "c6d1aa77984154bf72bbca3ac30b6a092379ecd7fdc6d2ddc1addc0b6f5f00b5"
+  end
+
+  resource "Module::Build" do
+    url "https://cpan.metacpan.org/authors/id/L/LE/LEONT/Module-Build-0.4229.tar.gz"
+    sha256 "1fe491a6cda914b01bc8e592faa2b5404e9f35915ca15322f8f2a8d8f9008c18"
+  end
+
+  # runtime deps, see: https://github.com/trizen/youtube-viewer/blob/master/META.json#L34-L53
+  resource "Data::Dump" do
+    url "https://cpan.metacpan.org/authors/id/G/GA/GAAS/Data-Dump-1.23.tar.gz"
+    sha256 "af53b05ef1387b4cab4427e6789179283e4f0da8cf036e8db516ddb344512b65"
+  end
+
+  resource "Encode" do
+    url "https://cpan.metacpan.org/authors/id/D/DA/DANKOGAI/Encode-3.01.tar.gz"
+    sha256 "d4555f570491648dbbd602bce7966672834b4c8f45acaa6757de474fca3a4d87"
+  end
```

Comment:
```ruby
end unless OS.linux?
```
I concern what all Linux users (not only Ubuntu-users) have Encode installed.
But it is just suggestion I don't understand anything at all about ~/.cpan ~/perl5 or local:lib, after digging homebrew's perl, plenv, some reading (about Pod)... The next error I got after skipping Encode is XSAWYERX/PathTools.